docs/integrations/amazon-aws/ses.md

```diff
@@ -352,7 +352,7 @@ See the details of delivered notifications including the email destinations, dom
 
 See the details of bounced notifications by email addresses, domains, bounce types, and bounce subtypes.
 
-<img src={useBaseUrl('img/integrations/amazon-aws/AmazonSESBouncedNotifications.png')} alt="AWS API Gateway" />
+<img src={useBaseUrl('img/integrations/amazon-aws/AmazonSESBouncedNotifications.png')} alt="AWS SES Bounced Notifications" />
 
 **Top Bounced email Addresses.** See the top 10 bounced email addresses by count in the last 24 hours.
```

Comment:
```suggestion
**Top Bounced email addresses.** See the top 10 bounced email addresses by count in the last 24 hours.
```

Reply:
<img width="593" alt="Screenshot 2023-06-09 at 4 28 39 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/88306913/940a867a-1084-42dc-98f9-2dab8ffd1fec">
It is expected here.